src/index.html

```diff
@@ -9,6 +9,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a class="is-active mark" href="#!">Apple</a>
+          </li>
+          <li class="nav__link">
+            <a class="link" href="#!">Samsung</a>
+          </li>
+          <li class="nav__link">
+            <a class="link" href="#!">Smartphones</a>
+          </li>
+          <li class="nav__link">
+            <a
+              class="link mark"
+              href="#!"
+              data-qa="hover">Laptops & Computers</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__link">
            <a class="link" href="#!">Smartphones</a>
          </li>

          <li class="nav__link">
            <a
              class="link mark"
              href="#!"
              data-qa="hover">Laptops & Computers</a>
          </li>
```

add blank line between multiple sibling blocks